larpandoracontent/LArUtility/RANSAC/PlaneModel.cc

```diff
@@ -0,0 +1,115 @@
+/**
+ *  @file   larpandoracontent/LArUtility/RANSAC/PlaneModel.cc
+ *
+ *  @brief  Implementation file for the PlaneModel, to be used in RANSAC.
+ *
+ *  $Log: $
+ */
+
```

Comment:
Probably doesn't make sense to be with RANSAC? Not 100%, its a RANSAC model, but its also specific to 3D hits since it uses protohits.